### DISCUSSION TITLE
Plugin similar to the 'Ammo' plugin but for Mage Spells

### DISCUSSION THREAD

**Discussion by Hannay39** [runelite/runelite#15932] [Ideas]
When using autocast with a staff it would be nice to have a little display to show how many casts of the selected spell you can do with your current runes (irrespective of charges). Useful for avoiding running out of runes mid-slayer tasks and probably useful for lots of wilderness activities to avoid risking too many runes. Probably work very similarly to the range 'ammo plugin'.

I'm mainly asking as this is a plugin I wouldn't mind making myself but wanted to check that this functionality doesn't exist anywhere currently or if it would be better suited within an existing plugin.

**Comment by benjaminlwest** [comment]
I like this idea - Would this include tracking powered staves? I have been on many a raid where someone mutters 'uh-oh' and their trident is out of charges.

 Could be useful for blowpipe/bowfa as well regarding scale/charges (this may be available, I don't use the ammo plugin personally)

**Reply by geheur** [reply]
I made a plugin for tracking powered stave charges ("weapon charges").
